.github/workflows/backend-workflow.yml

```diff
@@ -23,4 +23,4 @@ jobs:
         uses: cclauss/GitHub-Action-for-pylint@master
         with:
         # Installs Django and pylint-django, then runs pylint on all of the files in autoscheduler/
-          args: pip install -r .github/lint-requirements.txt ; pylint autoscheduler/** --load-plugins pylint_django
+          args: pip install -r .github/lint-requirements.txt ; pylint autoscheduler/scraper autoscheduler/scheduler autoscheduler/autoscheduler autoscheduler/manage.py --load-plugins pylint_django

```

Comment:
Doing `autoscheduler/**` lints non-Python files, such as `pytest.ini` or the README's, so we have to specify each individual directory.

We also can't just do `autoscheduler/`, since it doesn't have an `__init__.py`